.circleci/config.yml

```diff
@@ -24,16 +24,16 @@ jobs:
           keys:
             - v1-dependencies-{{ checksum "package.json" }}
             - v1-dependencies-
+      - run:
+          name: yarn install and dependencies
```

Comment:
nice, glad this worked 👌

Reply:
you can probably move this back down and you won't run into the caching issue.

Reply:
nvm.